Modules/Numerics/Optimizersv4/include/itkConjugateGradientLineSearchOptimizerv4.h

```diff
@@ -94,8 +97,9 @@ class ITK_TEMPLATE_EXPORT ConjugateGradientLineSearchOptimizerv4Template
   PrintSelf(std::ostream & os, Indent indent) const override;
 
 private:
-  DerivativeType m_LastGradient{};
-  DerivativeType m_ConjugateGradient{};
+  DerivativeType                m_LastGradient{};
+  DerivativeType                m_ConjugateGradient{};
+  TInternalComputationValueType m_InitialLearningRate{};
```

Comment:
Would it be an option to directly initialize `m_InitialLearningRate` to `1` here? As follows:

    TInternalComputationValueType m_InitialLearningRate{ 1 };

Reply:
Please, add a getter macro to `m_InitialLearningRate`. Exercise it in the test.

Reply:
Thanks, I added set / get macros for m_InitialLearningRate, and exercise them in the tests.